src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/TokenBasedFormattingRule.cs

```diff
@@ -371,6 +371,7 @@ private static SyntaxList<UsingDirectiveSyntax> GetUsings(SyntaxNode node)
 
             // * [
             if (currentToken.IsKind(SyntaxKind.OpenBracketToken) &&
+                !currentToken.Parent.IsKind(SyntaxKind.AttributeList) &&
```

Comment:
what happens if i have`[Foo] [Bar]` now?  how did it format before, how does it format now?

Reply:
Good question. Before and after behavior is that we leave whatever spaces you put there.
Should we make that 1 space?

Reply:
I went with 0 space as at least one existing test had that expectation.

Reply:
Yes.  0 spaces would be my expectation.